scripts/custom_lint_checks.py

```diff
@@ -216,19 +216,19 @@ class DocstringParameterChecker(checkers.BaseChecker):
 
     name = 'parameter_documentation'
     msgs = {
-        'W9005': ('''"%s" has constructor parameters
-                    documented in class and __init__''',
+        'W9005': (""""%s" has constructor parameters
```

Comment:
Is there a reason we can't use single quotes here, rather than triple double quotes? The lines here are showing up in GitHub as red.

Reply:
If we want to use single quotes here we probably need to disable pylint for this line.

The lines are also red with the single quotes, though, if you look at the code on the left hand side panel.

Reply:
Oh sorry, I meant one single quote rather than three. I'm not sure why this has to be a multiline docstring, it seems weird that the spaces at the start of line 220 are important.

Reply:
Done.

@seanlip I use single quotes for those lines.